honeycomb-core/src/twomap.rs

```diff
@@ -76,7 +76,7 @@ const TWO_MAP_BETA: usize = 3;
 /// The following example goes over multiple operations on the mesh in order
 /// to demonstrate general usage of the structure and its methods.
 ///
-/// ![TWOMAP_EXAMPLE](link_to_example_file.svg)
+/// ![TWOMAP_EXAMPLE](https://lihpc-computational-geometry.github.io/honeycomb/images/TwoMapExample.svg)
```

Comment:
Can you use relative link here?

Reply:
addressed in 67f5e97ff708f3e272f99a2bbfc79e645fa7aab4. It works locally, we'll need to check the deployed version.